src/launcher/util/appCompatibilityWarning.tsx

```diff
@@ -0,0 +1,292 @@
+    // According to https://docs.nordicsemi.com/bundle/nrfutil/page/guides/installing.html#prerequisites
+    if (semver.lt(device, '2.0.0')) {
+        return 'V7.80c';
+    }
+
+    if (semver.lt(device, '2.1.0')) {
+        return 'V7.88j';
+    }
+
+    if (semver.lt(device, '2.5.4')) {
+        return 'V7.94e';
+    }
+
+    return 'V7.94i';
+};
+
+const checkJLinkRequirements: AppCompatibilityChecker = async (
+    app: LaunchableApp
+) => {
+    if (!isDownloadable(app)) {
+        return undecided;
+    }
+
+    const deviceVersion =
+        app.versions?.[app.currentVersion]?.nrfutilModules?.device?.at(0);
+
+    if (!deviceVersion) {
+        return undecided;
+    }
+
+    const userDir = getUserDataDir();
+    const sandbox = await getSandbox(userDir, 'device', deviceVersion);
+    const moduleVersion = await sandbox.getModuleVersion();
+    const jlinkVersionDependency = resolveModuleVersion(
+        'JlinkARM',
+        moduleVersion.dependencies
+    );
+
+    if (!jlinkVersionDependency) {
+        const requiredVersion = nrfutilDeviceToJLink(deviceVersion);
+
+        return incompatible(
+            'Missing dependency',
+            `Required SEGGER J-Link not found, expected version ${requiredVersion}`,
+            <div className="tw-flex tw-flex-col tw-gap-2">
+                <div>This app requires SEGGER J-Link {requiredVersion}.</div>
+                <div>
+                    <Link href="https://www.segger.com/downloads/jlink/">
+                        Download
+                    </Link>{' '}
+                    and install the SEGGER J-Link J-Link Software and
+                    Documentation pack {requiredVersion}. Restart nRF Connect
+                    for Desktop afterwards.
+                </div>
+            </div>,
+            {
+                warningKind: WarningKind.JLINK,
+                app: app.name,
+                nrfutilDevice: deviceVersion,
+                requiredJlink: requiredVersion,
+                actualJlink: 'none',
+            }
+        );
+    }
+
+    if (
+        jlinkVersionDependency.expectedVersion &&
+        existingIsOlderThanExpected(jlinkVersionDependency)
+    ) {
+        const expectedVersionNumber = strippedVersionName(
+            jlinkVersionDependency.expectedVersion
+        );
+        const actualVersionNumber = strippedVersionName(jlinkVersionDependency);
+
+        return incompatible(
+            'Outdated dependency',
+            `Untested version V${actualVersionNumber} of SEGGER J-Link found, ` +
+                `expected at least version V${expectedVersionNumber}`,
+            <div className="tw-flex tw-flex-col tw-gap-2">
+                <div>
+                    This app was tested with SEGGER J-Link V
+                    {expectedVersionNumber} but version V{actualVersionNumber}{' '}
+                    was found on your system.
+                </div>
+                <div>Things might not work as expected!</div>
```

Comment:
```suggestion
                <div>The apps might not work as expected!</div>
```

Reply:
Since the dialog comes up when trying to launch a single app, I will make this a single “The app might not …”.